app/scripts/migrations/088.ts

```diff
@@ -59,7 +59,11 @@ function migrateData(state: Record<string, unknown>): void {
 
           if (isObject(nftContractsByChainId)) {
             for (const chainId of Object.keys(nftContractsByChainId)) {
-              if (chainId === 'undefined' || chainId === undefined) {
+              if (
+                chainId === 'undefined' ||
+                chainId === undefined ||
```

Comment:
Pretty sure `Object.keys` would always return `undefined` as a string literal, do we shouldn't need this condition. But it was already there, so I left it to minimize changes.

I confirmed that using `null` here didn't work, I needed to check for the string `'null'`.